src/dev/i18n/__fixtures__/extract_default_translations/test_plugin_2/test_file.jsx

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
should the license be 
```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */
```

Reply:
Good catches! I am not sure. I grabbed the license from: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/dev/i18n/__fixtures__/extract_default_translations/test_plugin_1/test_file_1.jsx#L1-L10

Do we have a standard that should be used?

Reply:
EsLint should automatically add this for you if you have it configured correctly in your editor. But to answer your question, the License @ananzh provided is the standard :)